packages/foam-vscode/src/test/suite-unit.ts

```diff
@@ -8,32 +8,25 @@
  *   they will make direct use of the vscode API to be invoked as commands, create editors,
  *   and so on..
  */
+
+process.env.FORCE_COLOR = '1';
+process.env.NODE_ENV = 'test';
+
 import path from 'path';
 import { runCLI } from '@jest/core';
 
 const rootDir = path.join(__dirname, '..', '..');
 
 export function runUnit(): Promise<void> {
-  process.env.FORCE_COLOR = '1';
-  process.env.NODE_ENV = 'test';
-  process.env.BABEL_ENV = 'test';
-
   return new Promise(async (resolve, reject) => {
     try {
       const { results } = await runCLI(
         {
           rootDir,
           roots: ['<rootDir>/src'],
-          transform: JSON.stringify({ '^.+\\.ts$': 'ts-jest' }),
           runInBand: true,
           testRegex: '\\.(test)\\.ts$',
           setupFiles: ['<rootDir>/src/test/support/jest-setup.ts'],
-          setupFilesAfterEnv: ['jest-extended'],
-          globals: JSON.stringify({
-            'ts-jest': {
-              tsconfig: path.join(rootDir, 'tsconfig.json'),
-            },
-          }),
```

Comment:
The ts-jest related settings doesn't seem to be necessary, the official documentation suggests to just set the preset to ts-jest. I did that and found no issue.